cluster/gce/gci/configure-helper.sh

```diff
@@ -1985,6 +1985,10 @@ function run-kube-controller-manager-as-non-root {
 #   CLOUD_CONFIG_MOUNT
 #   DOCKER_REGISTRY
 function start-kube-controller-manager {
+  if [[ "${KUBE_CONTROLLER_MANAGER_CRP:-}" == "true" ]]; then
```

Comment:
I wonder if this might be better handled as a call out to is KCM install enabled check in internal-configure-helper.sh.

Reply:
Done.